src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -361,4 +411,14 @@ private void handleImport(Map<String, Object> map) {
             plugin.addChatSuccess("The following settings were merged (rather than being overwritten): " + String.join(", ", mergedConfigs));
         }
     }
+
+    static {
+        PROBLEMATIC_VARBITS = ImmutableSet.of(
+            KillCountNotifier.KILL_COUNT_SPAM_FILTER,
+            CombatTaskNotifier.COMBAT_TASK_REPEAT_POPUP,
+            Varbits.COLLECTION_LOG_NOTIFICATION,
+            PetNotifier.LOOT_DROP_NOTIFICATIONS,
+            PetNotifier.UNTRADEABLE_LOOT_DROPS
+        );
```

Comment:
This is only used for `contains` lookup; should we avoid `Set` overhead by just creating a helper method like so
```java
public static boolean isProblematicVarbit(int id) {
    return id == X || id == Y || id == Z;
}
```

Reply:
I think this solution is fine